index.html

```diff
@@ -1,31 +1,71 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
+  <head>
     <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Library of Congress Search Tool</title>
     <!-- Latest compiled and minified CSS -->
-    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" crossorigin="anonymous">
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+      integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC"
+      crossorigin="anonymous"
+    />
 
-    <link rel="stylesheet" href="./assets/css/style.css">
-    
-</head>
-<body>
-    <header>
-    </header>
-    <main class="row layout">
-        <div class="col-3">
-            <div id="search-form">
-            </div>
+    <link rel="stylesheet" href="./assets/css/style.css" />
+  </head>
+  <body class="d-flex align-content-center">
+    <main class="container">
+        <div class="row vh-100 justify-content-center align-items-center">
+      <form class="col-lg-10" id="formInputs">
+        <h1 class="text-center text-light" id="formTitle">Library of Congress Search Engine</h1>
+        <br>
+        <div class="form-group">
+          <input
+            type="text"
+            class="form-control form-control-lg"
+            id="searchInput"
+            placeholder="Search"
+          />
         </div>
-        <div class="col-9">
-            <div id="search-results"></div>
+        <br>
+        <div class="form-group">
+          <select class="form-control form-control-lg" id="formatInput">
+            <option>Select a Format</option>
+            <option value="maps">Maps</option>
+            <option value="audio">Audio</option>
+            <option value="photos">Photos</option>
+            <option value="amuscripts">Manuscripts</option>
+            <option value="newspapers">Newspapers</option>
+            <option value="film-and-videos">Film & Video</option>
+            <option value="notated-music">Notated Music</option>
+            <option value="websites">Websites</option>
+          </select>
         </div>
+        <br>
```

Comment:
IS there a reason you chose to use line breaks instead of setting the margin for the input boxes?

Reply:
I started with breaks before trying margins, I can try margins instead if that would be preferred

Reply:
Margins just give more control over the spacing between elements. For now it is fine, mainly wanted to test this feature